PetaLinux 2024.1, petalinux-create Permission Denied Error

Running `202511121-0` PetaLinux 2024.1 with Podman, the image also built by Podman.

`petalinux-create` gives permission denied error.

Here is the full log:

```text
ebox@517f159d77c3:~$ petalinux-create -n test project
[INFO] Create project: test
[ERROR] [Errno 13] Permission denied: '/home/ebox/test/project-spec/meta-user/README'

ebox@517f159d77c3:~$ printenv | grep -i ebox
EBOX_OCI_TITLE=EBox - PetaLinux 2024.1
PWD=/home/ebox
EBOX_OCI_VERSION=20251122-0
HOME=/home/ebox
EBOX_OCI_REVISION=202511121-0
EBOX_OCI_AUTHORS=The EBox Authors
EBOX_OCI_LICENSE=MIT
EBOX_OCI_DESCRIPTION=PetaLinux 2024.1 in a container, the EBox project
```

Suspicious line:

https://github.com/alperyazar/EBox/blob/c62ec736f7387db12acd5c9bd46aeb6a82b0e779/xilinx/petalinux/2024.1/Dockerfile#L175

No problem for existing projects, can be built by Podman.

---

I think no need to be a paranoid about restricting write permission, can remove this.

--- a/xilinx/petalinux/2024.1/Dockerfile
+++ b/xilinx/petalinux/2024.1/Dockerfile
@@ -209,7 +209,7 @@ RUN /opt/EBox/bin/replace_sources.sh /opt/EBox/sources-final.list
 ARG EBOX_OCI_TITLE="EBox - PetaLinux 2024.1"
 ARG EBOX_OCI_DESCRIPTION="PetaLinux 2024.1 in a container, the EBox project"
 ARG EBOX_OCI_VERSION="19700101-0"
-ARG EBOX_OCI_REVISION="20251211-DRAFT-0"
+ARG EBOX_OCI_REVISION="20251212"
 ARG EBOX_OCI_LICENSE="MIT"
 ARG EBOX_OCI_AUTHORS="The EBox Authors"
 

